bottles/backend/utils/manager.py

```diff
@@ -224,6 +231,7 @@ def create_desktop_entry(
         skip_icon: bool = False,
         custom_icon: str = "",
         use_xdp: bool = False,
+        window: Gtk.Window = None,
```

Comment:
The type signature is lying. Obviously it is an `Optional[Gtk.Window]`.

What is this window used for? Consider renaming the variable.